templates/input_website.html

```diff
@@ -4,15 +4,74 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sale Tracker</title>
+    <style>
```

Comment:
Great format! Maybe we can add enticing images along the side, but this is more of an extension thing.